android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -450,6 +452,18 @@ public void setDebug(final ReactExoplayerView videoView,
         }
     }
 
+    @ReactProp(name = PROP_CONTROLS_STYLES)
```

Comment:
small mistake here !

Reply:
What is mistake?

Reply:
@seyedmostafahasani The function is declared twice (in code comment doesn't highlight this !)